reframe/frontend/statistics.py

```diff
@@ -281,6 +281,17 @@ def print_failure_stats(self, printer):
             for f in l[1:]:
                 stats_body.append(row_format.format('', '', str(f)))
 
+        for p, l in failures.items():
+            stats_body.append(stats_hline)
```

Comment:
I am not sure whether we want a `stats_hline` here or a solid line. The result now is:
```
==============================================================================
FAILURE STATISTICS

Total number of test cases: 10
Total number of failures: 4

Phase         #     Failing test cases
------------- ----- ------------------------------------------------------------
sanity        1     [T2, builtin, generic:default]
startup       2     [T7, None, None]
                    [T9, None, None]
setup         1     [T8, builtin, generic:default]
------------- ----- ------------------------------------------------------------
Rerun the failed tests with:
 -n T2 -p builtin --system generic:default -r
------------- ----- ------------------------------------------------------------
Rerun the failed tests with:
 -n T7 -p None --system None -r
 -n T9 -p None --system None -r
------------- ----- ------------------------------------------------------------
Rerun the failed tests with:
 -n T8 -p builtin --system generic:default -r
------------------------------------------------------------------------------
```